notebooks/data-sources/oc-github-repo/github_PR_EDA.ipynb

```diff
@@ -283,7 +283,7 @@
     "\n",
```

Comment:
<p>Please just combine this with the cell above and make sure the statements remain consistent with the correlation chart. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/289/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>